RELEASENOTES.docu

```diff
@@ -1721,4 +1721,9 @@ extern typedef struct { } my_type_t;</pre> </add-note></build-id>
   <build-id value="6205"><add-note> Fixed a bug with how template types were
       handled by DMLC that could cause the compiler to still exhibit
       nondeterministic behaviour.</add-note></build-id>
+  <build-id value="next"><add-note>Fixed an issue where <tt>loggroup</tt>
+      declarations in a device would erroneously and silently cause the 32:nd
+      log group and beyond to not function properly. A DML device may now have
+      up to 63 log groups (61 excluding built-in log groups) without issue
+      <bug number="22018374443"/>.</add-note></build-id>
```

Comment:
Fun question: now that the original issue turned out to be completely unrelated, and (mostly) user error, should the releasenote still reference it?

Reply:
If the change is not user-visible in practice, then no release note is needed. The stakeholders of that ticket don't need a notification of this change, because they are unaffected by this package upgrade.